### PR TITLE
Fix ESLint Errors

### PR DESCRIPTION
**Pull request by JonColman94**
- [x] You have read the [Odin Bot README/Contributing Guide](https://github.com/TheOdinProject/odin-bot-v2/blob/master/README.md).

#### 1.Describe the changes made:

As the title suggests, all of the ESLint Errors have been fixed. Most of these were trivial semicolon/trailing comma errors. Other, less trivial errors have been amended to remain as true to the original code as possible. 

I have been made aware that some commands will need changes; and this commit has not made any substantial changes, outside of ESLint compliance.

#### 2. If this PR modifies a command, please provide a screenshot of the passing tests below. NOTE: Your PR will not be merged if you have any failing test cases. 

Screenshot from 2021-03-04 18-50-37
Screenshot from 2021-03-04 18-51-29
Screenshot from 2021-03-04 18-51-52
Screenshot from 2021-03-04 18-52-05
Screenshot from 2021-03-04 18-52-38
Screenshot from 2021-03-04 18-53-16
Screenshot from 2021-03-04 18-53-32
Screenshot from 2021-03-04 18-53-56
Screenshot from 2021-03-04 18-54-19
Screenshot from 2021-03-04 18-54-45
Screenshot from 2021-03-04 18-54-48
Screenshot from 2021-03-04 18-55-17
Screenshot from 2021-03-04 18-55-44
Screenshot from 2021-03-04 18-55-56

#### 3. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#116

### CODE REVIEW

File: .gitignore
Comment (twalton83) [inline .gitignore:4]: Any reason for this?

**Review by twalton83 — APPROVED** [review]
Looks good to me so far, I will defer to another maintainer/core member before merging because this is such a large change.

File: .gitignore
Comment (JonColman94) [reply .gitignore:4]: Oh, I overlooked that apparently. 

.commitmsg was purely so I could edit my commits while I made each change, and .env.sample was just me paranoidly making sure I didn't push any changes I might've made there.

Neither of them are critically important.

Comment (twalton83) [reply .gitignore:4]: Awesome, thank you, go ahead and remove this bit since it's not relevant.

Comment (JonColman94) [reply .gitignore]: Done

File: .env.sample
Comment (KevinMulhern) [inline .env.sample:1]: was removing this file intentional?

**Review by KevinMulhern — COMMENTED** [review]
Thanks for taking the time to do this @JonColman94, I just have one question.

File: .env.sample
Comment (JonColman94) [reply .env.sample:1]: Nope. I had renamed .env.sample to .env. not entirely sure why it got deleted. Will need to fix.

Good catch.

Comment (JonColman94) [reply .env.sample:1]: There. That should be that screwup fixed.

**Comment by KevinMulhern** [comment]
Hey @JonColman94, the .gitignore file and .env.sample are still showing up in the files changed in this PR.

If you've changed files in a branch unintentionally and want to get them back to the state they were in the master/main branch its easier to checkout the master version of those files:

`git checkout master .gitignore`
`git checkout master .env.sample`

Then commit those and push them up.

If you have any questions don't hesitate to let me know.

**Comment by JonColman94** [comment]
Okay, that should be it. Neither file is in the changed files anymore from what I can see.

**Review by marvingay — APPROVED** [review]

**Comment by JonColman94** [comment]
The latest fix as of [this](https://github.com/TheOdinProject/odin-bot-v2/pull/118/commits/aaf95f4825c5f75896ad03f397b4f02af4402f48) should pass all checks.

**Pull request merged**
